docs/r2dbc.md

```diff
@@ -0,0 +1,284 @@
+
+
+# Connecting to SQL Server using R2DBC
+
+## Setup and Usage
+
+### Adding the library as a dependency
+
+
+<!-- {x-release-please-start-version} -->
+
+#### Maven
+Include the following in the project's `pom.xml`:
+
+##### Mysql
+```maven-pom
+    <dependency>
+      <groupId>com.google.cloud.sql</groupId>
+      <artifactId>cloud-sql-connector-r2dbc-mysql</artifactId>
+      <version>1.13.1</version>
+    </dependency>
+```
+
+##### Postgres
+```maven-pom
+    <dependency>
+      <groupId>com.google.cloud.sql</groupId>
+      <artifactId>cloud-sql-connector-r2dbc-postgres</artifactId>
+      <version>1.13.1</version>
+    </dependency>
+```
+##### SQL Server
+```maven-pom
+    <dependency>
+      <groupId>com.google.cloud.sql</groupId>
+      <artifactId>cloud-sql-connector-r2dbc-sqlserver</artifactId>
+      <version>1.13.1</version>
+    </dependency>
+```
+
+#### Gradle
+Include the following the project's `build.gradle`
+
+##### Mysql
+
+```gradle
+compile 'com.google.cloud.sql:cloud-sql-connector-r2dbc-mysql:1.13.1'
+```
+*Note: Also include the R2DBC Driver for MySQL, `io.asyncer:r2dbc-mysql:<LATEST-VERSION>`
```

Comment:
Please make the `Note` markdown consistent in this doc. Currently there are:

```
*Note*:
Note:
**Note**:
```

Reply:
Fixed. I'm using `**Note:**`